### ISSUE TITLE
extended URL to point to gamecode

### ISSUE THREAD

**Issue by arrethra** [Lurkars/gloomhavensecretariat#594]
### Is your feature request related to a problem? Please describe.

My phone/browser eats cookies for breakfast (cookiemnster), i.e. it doesn't remember cookies. So everytime I would have to lookup/copy the gamecode and re-enter it.

### Describe the solution you'd like

Is it possible that urls can be generated to open a specific server with game-code in an instance. This URL can be saved within my bookmark with ease, and revisited in a simple click . 
such as https://gloomhaven-secretary.de/server=german?gamecode=1234-5678-90ab-cdef?/ 

Thnx
Arrethra

### Describe alternatives you've considered

alternative could be that I would install a browser that saves cookies, but I don't exactly like that 'workaround' (cookies are too yummy to not be eaten/destroyed)

**Comment by Lurkars** [comment]
I am not sure about your setup. I don't use any cookies at all. All data is stored in [IndexedDB](https://en.wikipedia.org/wiki/Indexed_Database_API) and if this is not available [Local Storage](https://en.wikipedia.org/wiki/Web_storage) is used as fallback. If both wouldn't be available due to extensions or special browser settings the whole app would just not work for you. So at least, it is working and not storing it in long term means, that after your session you (or an extension) clears the storage of the page somehow. So since the app is general working and the server connection is just a setting, I don't like to add a very specialized feature to encode settings for users using a setup not made for this app.

So to sum up: I am using a cookie blocker too and the app is working fine. So I would first recommend that you are checking your settings. You can accept to store indexeddb/local storage data and still block all cookies. If you're using any private mode, then yes no data is stored. As said, I won't build work around for specialized setups.
